readme.md

```diff
@@ -670,6 +670,7 @@
 - [Cytodata](https://github.com/cytodata/awesome-cytodata#readme) - Image-based profiling of biological phenotypes for computational biologists.
 - [IRC](https://github.com/davisonio/awesome-irc#readme) - Open source messaging protocol.
 - [Advertising](https://github.com/cenoura/awesome-ads#readme) - Advertising and programmatic media for websites.
+- [Climate Action](https://github.com/philsturgeon/awesome-earth#readme) - Find ways to resolve the climate crisis.
```

Comment:
The title should be the awesome list name.

Reply:
Fixed